.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  # Watch dependencies for current stable
+  - package-ecosystem: "bundler"
+    directory: "/docker-image/v1.13"
```

Comment:
We might forget to upgrade this...

Reply:
So I prefer black list instead of white list.
(Move obsolete versions to other directory?)

Reply:
Ah, moving obsolete versions is better, I mean that set `directory: "/docker-image/"` and move docker-image/v0.12 to somewhere.